inspire_dojson/hep/rules/bd7xx.py

```diff
@@ -195,17 +208,23 @@ def related_records_78708(self, key, value):
 
 @hep2marc.over('78002', '^related_records$')
 @utils.for_each_value
-def related_records2marc(self, key, value):
+def related_preceding_records2marc(self, key, value):
```

Comment:
Name shouldn't change here.

Reply:
:+1: it also makes sense to populate `78708` instead of `78002` (as this contains the other relations) and do the `78002`/`78502` through side-effects to make it more symmetric between predecessor/successor, but it's not crucial.

Reply:
Ah yes